dashboard/app/views/levels/editors/_blockly.html.haml

```diff
@@ -1,3 +1,20 @@
+- content_for(:head) do
+  = javascript_include_tag 'js/blockly'
+  - if @level.game && @level.game.app
+    = javascript_include_tag "js/en_us/blockly_locale"
+    = javascript_include_tag "js/en_us/common_locale"
+    = javascript_include_tag "js/en_us/#{@level.game.app}_locale"
+    :javascript
+        Blockly.Css.styleSheet_ = goog.cssom.addCssText(Blockly.Css.CONTENT.join("\n")).sheet
+        Blockly.assetUrl = function (path) {
+          return '/assets/' + path;
+        };
+
+        levelbuilder.installBlocks("#{@level.game.app}", Blockly, {
+          skin: {id: "#{@level.skin}"},
+          isK1: "#{@level.try(:isK1)}",
+        });
```

Comment:
not so much copied as just manually doing the bare minimum necessary to enable just the blockly functionality we want on a page that has nothing. I'm not sure it's worth extracting.

Or rather, it might be worth extracting, but doing so correctly would probably end up being more work than it's worth, and if we're going to do something incorrectly I'd just as soon it be obviously incorrect like this implementation.